### ISSUE TITLE
Week 4 - Node Girls workshop

### ISSUE THREAD

**Issue by ZooeyMiller** [foundersandcoders/master-reference#781]
We currently use the [node-girls node-workshop](https://github.com/node-girls/node-workshop) which, I think is a fantastic introduction to creating a CMS/writing a server, after the code along on week 4. 

I remember in N2, and now in G3, we have had problems with this workshop due to the fact it uses [jQuery for the api call to the server](https://github.com/node-girls/node-workshop/blob/master/public/script.js). Due to the fact this is the only example in node week in a workshop/code along of communicating with a server with AJAX (other than the modularisation challenge), the fact it is in jQuery means that students are sometimes confused with:
1. how to communicate with a their own server with AJAX (sometimes it just doesn't click that they need to use an XHR). 
2. What is even happening in the example.

There is also a problem with the naming conventions, in all of the examples we give of servers, the file which handles the routes is called `router.js`, and the file which actually contains the functionality of _how_ the routes are handled is called `handler(s).js`. In the node-girls workshop the equivalent file to `router.js` is `handler.js`. Students, rightly, find this confusing.

It may seem odd that I am raising this issue here rather than in the node-girls/node-workshop repo, however my reasoning is that neither of these things are inherently problematic. Using jQuery for AJAX and having a different naming convention to what we predominantly use at FAC is fine, the problem is not with the workshop itself, but with the workshop as it is used in the FAC curriculum. I don't think it's necessarily appropriate to raise issues on a different organisations repo, with requests just so it fits better into our curriculum. 

However, I am left with a question of how to proceed? The workshop in general is pretty great, and I don't think we should reinvent the wheel when it comes to these workshops, since these minor things are the only problem. Do we ask node-girls if they are ok with us forking to change these things?

did other campuses/cohorts have similar issues?

**Comment by astroash** [comment]
@ZooeyMiller I agree with everything you say here, London had exactly the same problem. This came up in [our SGC](https://github.com/foundersandcoders/london-programme/blob/master/stop-go-continue/fac-12/week4.md#groups-getting-too-far-apart-mon--tues-workshops-hannah), and students requested whether we could fork this repo for our own use in the future.

**Comment by ZooeyMiller** [comment]
@astroash It's something to potentially to consider, but not something I think we should make a decision on now. I'd like people's thoughts in general though, and when L12, N3 and G3 are curriculum planning they can take this into consideration.

**Comment by claireinez** [comment]
Hey @ZooeyMiller, some interesting points! We've discussed it at Node Girls (cc @msmichellegar and @minaorangina) and we'd be happy to work with you all to make it better for FAC (we don't use the pure node workshop that much in our public-facing workshops any more, and obvs we're all FAC alumnae so we'd love to help out). Changing the file names is absolutely fine and low effort, and we'd be open to dropping jQuery in favour of XHR. Maybe we could catch up in person sometime and work to update it?

**Comment by ZooeyMiller** [comment]
@claireinez That's fantastic! I'm back in London next week so let's sort meeting up and working on it when I'm back.

**Comment by astroash** [comment]
@claireinez would be awesome if you'd be able to check out this PR ❤️
https://github.com/node-girls/node-workshop/pull/53
